labscript_devices/AD9959DDSSweeper/blacs_workers.py

```diff
@@ -0,0 +1,201 @@
+#####################################################################
+#                                                                   #
+# /labscript_devices/AD9959DDSSweeper/blacs_workers.py              #
+#                                                                   #
+# Copyright 2025, Carter Turnbaugh                                  #
+#                                                                   #
+# This file is part of the module labscript_devices, in the         #
+# labscript suite (see http://labscriptsuite.org), and is           #
+# licensed under the Simplified BSD License. See the license.txt    #
+# file in the root of the project for the full license.             #
+#                                                                   #
+#####################################################################
+
+from blacs.tab_base_classes import Worker
+import labscript_utils.h5_lock, h5py
+
+class AD9959DDSSweeperInterface(object):
+    def __init__(
+                self,
+                com_port,
+                sweep_mode,
+                timing_mode,
+                ref_clock_frequency,
+                pll_mult
+                ):
+        global serial; import serial
+
+        self.timeout = 0.1
+        self.conn = serial.Serial(com_port, 10000000, timeout=self.timeout)
+
+        version = self.get_version()
+        print(f'Connected to version: {version}')
+
+        board = self.get_board()
+        print(f'Connected to board: {board}')
+
+        current_status = self.get_status()
+        print(f'Current status is {current_status}')
+
+        self.conn.write(b'reset\n')
+        self.assert_OK()
+        self.conn.write(b'setclock 0 %d %d\n' % (ref_clock_frequency, pll_mult))
+        # self.assert_OK()
```

Comment:
I think this should be uncommented, and one of the two `self.assert_OK()`s two lines down removed.
It is probably buffering the result of the `setclock` instruction until after the `mode` instruction, which might be confusing in the future.